### PR TITLE
Update App.vue

### LINKED ISSUE #2: Suffix '@hypothes.is' to the value of user query parameter
The request is getting 304 response which results in 'no matching options' for user
Adding the suffix results in 200 with data
Line to fix - https://github.com/c6p/logseq-hypothesis/blob/25a76b7e190ed1b665710c2bb8f0a2ef6bfb3937/src/App.vue#L110

### PR DESCRIPTION
**Pull request by nickdex**
Fix empty response to the search annotation request

Resolves #2

### CODE REVIEW

**Comment by c6p** [comment]
@nickdex thanks for the PR. But hypothes.is [is not the only authority](https://web.hypothes.is/help/overview-of-the-hypothesis-system/), hence `c6p@hypothes.is` as an account. However, since 99% of the users will just use hypothes.is, I'm all for defaulting to it.

Could you modify your PR, so when user account does not contain `@`, it adds `@hypothes.is` else leaves it unmodifed?

**Comment by nickdex** [comment]
@c6p Ah okay, I was using it wrong then! 
Like you said, hypothes.is will be what most people so update the PR accordingly.

P.S. - Would you mind adding ['hacktoberfest-accepted' label](https://hacktoberfest.digitalocean.com/resources/participation) to this PR?

File: src/App.vue [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default {
         params: {
           limit: 200,
           order: 'asc',
-          user: `acct:${this.user}@hypothes.is`,
+          user: `acct:${this.user}${this.user.includes('@') ? '@hypothes.is' : '' }`,
```
Comment (c6p) [inline src/App.vue]: Ternary reversed, could you also fix that? If not has `@` add hypothes.is. Then all is fine.

Comment (nickdex) [reply src/App.vue]: Whoops! fixed it.

**Review by c6p — APPROVED** [review]
Thanks for the PR.

**Pull request merged**